website_event_questions_template/i18n/es.po

```diff
@@ -0,0 +1,185 @@
+msgid "Ask each attendee"
+msgstr "Preguntar a cada asistente"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_create_uid
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_create_uid
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_create_uid
+msgid "Created by"
+msgstr "Creado por"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_create_date
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_create_date
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_create_date
+msgid "Created on"
+msgstr "Creado en"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_display_name
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_display_name
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_display_name
+msgid "Display Name"
+msgstr "Nombre a mostrar"
+
+#. module: website_event_questions_template
+#: model:ir.model,name:website_event_questions_template.model_event_event
+msgid "Event"
+msgstr "Evento"
+
+#. module: website_event_questions_template
+#: model:ir.actions.act_window,name:website_event_questions_template.act_event_question_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_template_id_6171
+#: model:ir.ui.view,arch_db:website_event_questions_template.view_event_question_template_form
+msgid "Event Question Template"
+msgstr "Plantilla de preguntas de eventos"
+
+#. module: website_event_questions_template
+#: model:ir.ui.view,arch_db:website_event_questions_template.view_event_question_template_question_form
+msgid "Event Question Template Question"
+msgstr "Preguntas de la plantilla"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_id
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_id
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_id
+msgid "ID"
+msgstr "ID (identificación)"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,help:website_event_questions_template.field_event_question_template_question_is_individual
+msgid ""
+"If True, this question will be asked for every attendee of a reservation. If "
```

Comment:
You still continue using POEdit with the line wrapping option, which totally breaks the PO file. Please don't use it or remove that option.